airbyte-webapp/src/locales/en.json

```
@@ -268,6 +268,7 @@
   "sources.cancelled": "Cancelled",
   "sources.incomplete": "Retrying",
   "sources.partialSuccess": "Partial Success",
+  "sources.streamsReset": "{count, plural, =0 {0 streams} one {# stream} other {# streams}} reset",
```

From @andyjih's comment in:
https://github.com/airbytehq/airbyte/pull/14621/files#r929015290

```suggestion
  "sources.streamsReset": "{count, plural, =0 {No streams} one {# stream} other {# streams}} were reset",
```

---

fixed